docs/creating-presets.md

```diff
@@ -272,35 +272,42 @@ Set the main entry points for the application. If the option is not set, Neutrin
   index: 'index'
 }
 ```
- 
+
 Notice the entry point has no extension; the extension is resolved by webpack. If relative paths are specified,
 they will be computed and resolved relative to `options.source`; absolute paths will be used as-is.
- 
+
 By default these main files are not required to be in JavaScript format. They may also potentially be JSX, TypeScript,
 or any other preprocessor language. These extensions should be specified in middleware at
 `neutrino.config.resolve.extensions`.
 
+
```

Comment:
Are there 2 blank lines here?

Reply:
Ah yes I'll remove this additional newline